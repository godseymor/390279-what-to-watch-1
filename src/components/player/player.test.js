import React from 'react';
import renderer from 'react-test-renderer';

import Player from './player.jsx';

const mock = {
  movie: {
    title: `Fantastic Beasts: The Crimes of Grindelwald`,
    poster: `poster.jpg`,
    preview: `video.mp4`,
  },
};

it(`Player renders correctly`, () => {
  const props = {
    moviePreview: mock.movie.preview,
    moviePoster: mock.movie.poster,
    settings: {
      width: 280,
      height: 175,
    }
  };

  const tree = renderer.create(<Player {...props}/>).toJSON();

  expect(tree).toMatchSnapshot();
});
