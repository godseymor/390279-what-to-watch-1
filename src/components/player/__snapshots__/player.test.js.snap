// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player renders correctly 1`] = `
<video
  height={175}
  poster="poster.jpg"
  src="video.mp4"
  width={280}
/>
`;
